src/components/contentContainer.tsx

```diff
@@ -6,6 +6,11 @@ import { ContentContainerInformationFragment } from '../../types/graphql-types';
 import { fontStyles } from '../style/fonts';
 import ContentBlock from './contentBlocks/contentBlock';
 import Carousel from './contentContainer/carousel';
+import PetitionsContentContainer from './contentContainer/petitionsContentContainer';
+
+export enum ContentContainerAppearance {
```

Comment:
i'd probably use a type here rather than an enum

alternatively you could add Carousel to this type

Reply:
I extended the enum. Please review again.